buildspec_canary.yml

```diff
@@ -0,0 +1,14 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      golang: 1.13
+  build:
+    commands:
+      - GOPROXY=direct
```

Comment:
nit: can this be part of env vars? Also, I think this is only needed on Amazon internal network. Should technically able to call Go package registry on Codebuild.

Reply:
Yes, will change this in the CM step to update the canary's env vars.

Reply:
Removed in most recent commit.